frontend/smart-kickers-game/src/components/Game/GameStatistics/GameStatistics.js

```diff
@@ -21,6 +21,19 @@ function GameStatistics({ finalScores, onNewGameRequested }) {
     return Team === teamID ? Speed.toFixed(2) + ' km/h' : '😵';
   }
 
+  function getManualChangedGoals(teamID) {
+    if (!statistics?.ManualGoals) return;
+    return statistics.ManualGoals[teamID];
```

Comment:
Endpoint has changed.
```suggestion
    if (!statistics?.manualGoals) return;
    return statistics.manualGoals[teamID];
```